### ISSUE TITLE
Learning about Issues with a ReadMe

### ISSUE THREAD

**Issue by LadyPendragon** [LadyPendragon/hello-world#1]
What is the difference really between write and preview?  Will Preview go away once I write and submit the new issue?  Interesting.... I will try.

Yes!  It does!  I can Preview what will appear after I write and submit/update.... and then the Preview tab goes away.  Cool!

**Comment by lukel97** [comment]
Yep! Previews are there so you can check that your markup looks correct before you post it
